{% extends "pred/base_layout.html" %}

{%block content_right_panel %}
    {% if username %}
      {% include "pred/user_header.html" %}
    {% endif %}
  <h2>Help</h2>
  <table cellspacing=0 cellpadding=0 border=0>
    <tr>
      <td class="Greylink"><a href="#1">1.&nbsp;Summary<img src="{{STATIC_URL}}/images/0.gif" height=1 width=5 border=0><img src="{{STATIC_URL}}/images/arrow.png" border=0><img src="{{STATIC_URL}}/images/0.gif" height=1 width=40 border=0></a><td>
      <td class="Greylink"><a href="#2">2.&nbsp;Usage<img src="{{STATIC_URL}}/images/0.gif" height=1 width=5 border=0><img src="{{STATIC_URL}}/images/arrow.png" border=0><img src="{{STATIC_URL}}/images/0.gif" height=1 width=40 border=0></a><td>
      <td class="Greylink"><a href="#3">3.&nbsp;Output<img src="{{STATIC_URL}}/images/0.gif" height=1 width=5 border=0><img src="{{STATIC_URL}}/images/arrow.png" border=0><img src="{{STATIC_URL}}/images/0.gif" height=1 width=40 border=0></a><td>
      <td class="Greylink"><a href="#4">4.&nbsp;References<img src="{{STATIC_URL}}/images/0.gif" height=1 width=5 border=0><img src="{{STATIC_URL}}/images/arrow.png" border=0><img src="{{STATIC_URL}}/images/0.gif" height=1 width=40 border=0></a><td>
      <td class="Greylink"><a href="#5">5.&nbsp;Contact<img src="{{STATIC_URL}}/images/0.gif" height=1 width=5 border=0><img src="{{STATIC_URL}}/images/arrow.png" border=0><img src="{{STATIC_URL}}/images/0.gif" height=1 width=40 border=0></a><td>
      </tr>
  </table>

  <table>
    <tr>
      <td valign="top" colspan="2">
      <br><br>
      <a name="1"></a>
      <h3>1. Summary</h3><br>
      <table cellspacing=0 cellpadding=0 border=0>
        <tr>
          <td><img src="{{STATIC_URL}}/images/0.gif" border=0 height=1 width=400></td>
          <td><img src="{{STATIC_URL}}/images/0.gif" border=0 height=1 width=50></td>
          <p>
          Direct-coupling based contact prediction methods (DCA) enable de-novo
          structure prediction of proteins with an unprecedented accuracy.
          However, these methods require thousands of protein sequences to
          achieve high accuracy, limiting their usability. Here, we introduce
          PconsC3, a method to accurately predict contacts for families an
          order of magnitude smaller than feasible with DCA, thus increasing
          accurate contact predictions from 12% to 54% of all protein domain
          families with unknown structure.  Input features comprise contact
          predictions by plmDCA, GaussDCA as well as PhyCMAP, secondary
          structure prediction by PSIPRED 3.0, and solvent accessibility
          prediction by NetSurfP 1.1. In PconsC3 PhyCMAP can be replaced by
          another contact predictor and we have successfully used CMapPro with
          similar accuracy. Additionally, CD-HIT is run to generate statistics
          about the alignment (i.e. alignment depth at different sequence
          similarity cut-offs). The initial layer of PconsC3 takes these
          features as input and uses a random forest to predict a score for
          each possible contact. On contrary to previous work, PconsC3 applies
          pattern recognition already in the first layer. This results in an
          intermediate contact map. Every following layer uses all the initial
          features plus the output from the previous layer, given as a window
          of 11 by 11 residues around the current contact.
          </p>
         {% include "pred/egi_acknowledgement.html" %}

        </tr>
        <tr><td>
                <figure>
                <img src="{{STATIC_URL}}/images/PconsC3_workflow.png" border=0 width=400 >
                 <figcaption>Fig. 1: Workflow of PconsC3.</figcaption>
             </figure>
        </td> </tr>
      </table>

      <br><br>

      <a name="2"></a>
      <h3>2. Usage</h3><br>
      <table cellspacing=0 cellpadding=0 border=0>
        <tr>
          <td><img src="{{STATIC_URL}}/images/0.gif" border=0 height=1 width=400></td>
          <td><img src="{{STATIC_URL}}/images/0.gif" border=0 height=1 width=50></td>
        </tr>
        <tr><td>
            <p>
            Input to the server is one or several (upto five) amino acid sequence(s) in <a
              href='http://blast.ncbi.nlm.nih.gov/blastcgihelp.shtml'
              target=\'_blank\'>FASTA</a> format. The user can either paste
            sequences in the text-area provided, or, alternatively,
            upload a file containing your sequences.
            </p>
            <i>Example input:</i> <br>
            <span style="font-family: courier">
                &gt;1qj8A<br>
                ATSTVTGGYAQSDAQGQMNKMGGFNLKYRYEEDNSPLGVIGSFTYTEKSRTASSGDYNKN<br>
                QYYGITAGPAYRINDWASIYGVVGVGYGKFQTTEYPTYKNDTSDYGFSYGAGLQFNPMEN<br>
                VALDFSYEQSRIRSVDVGTWIAGVGYRF<br>

            </span>
            <br>
      </td></tr>
    </table>

    <br><br>

      <a name="3"></a>
      <h3>3. Output</h3><br>
      <table cellspacing=0 cellpadding=0 border=0>
        <tr>
          <td><img src="{{STATIC_URL}}/images/0.gif" border=0 height=1 width=400></td>
          <td><img src="{{STATIC_URL}}/images/0.gif" border=0 height=1 width=50></td></tr>
        <tr><td>
                <p>
                The server outputs the contact predictions using PconsC3
                method in plain text format. The predicted contacts are also
                displayed graphically. Additionally, a zipped folder including the
                input sequence, predicted contacts (plain text and graphical
                representation) and multiple sequence alignments is also downloadable.
                </p>
      </td></tr>
    </table>
    <br><br>

    <a name="4"></a>

    <h3>4. References</h3><br>
    <table cellspacing=0 cellpadding=5 border=0>
      <tr><td><img src="{{STATIC_URL}}/images/0.gif" border=0 height=1 width=400></td>
        <td><img src="{{STATIC_URL}}/images/0.gif" border=0 height=1 width=50></td>
      </tr>
     <tr><td>
        Skwark MJ, Michel M, Hurtado DM, Ekeberg M, Elofsson A. "Accurate contact predictions for thousands of protein families using PconsC3."
     </td></tr>

     <tr><td>
        Skwark MJ, Raimondi D, Michel M, Elofsson A (2014) "Improved Contact Predictions Using the Recognition of Protein Like Contact Patterns." PLoS Computational Biology 10(11).
     </td></tr>
  </table>

  <br><br>
  <a name="5"></a>

    <h3>5. Contact</h3><br>
    <table cellspacing=0 cellpadding=0 border=0>
      <tr>
        <td><img src="{{STATIC_URL}}/images/0.gif" border=0 height=1 width=400></td>
        <td><img src="{{STATIC_URL}}/images/0.gif" border=0 height=1 width=50></td>
      </tr>
      <tr><td>
          <b>Arne Elofsson group</b><br><br>
          <!--<a href="http://www.cbr.su.se/">Center for Biomembrane Research</a><br><br>-->
          <a href="http://www.dbb.su.se/">Department for Biochemistry and Biophysics</a><br>
          The Arrhenius Laboratories for Natural Sciences<br>
          <a href="http://www.su.se/">Stockholm University</a><br>
          SE-106 91 Stockholm, Sweden<br><br>

          <a href="http://www.scilifelab.se">Science for Life Laboratory</a><br>
            Box 1031, 17121 Solna, Sweden
            <br><br>
          <table cellspacing=0 cellpadding=0 border=0>
            <tr><td>E-mail:</td><td>&nbsp;&nbsp;&nbsp;</td><td><a href="mailto:arne@bioinfo.se">arne@bioinfo.se</a></td></tr>
            <tr><td>Phone:</td><td>&nbsp;&nbsp;&nbsp;</td><td>(+46)-8-16&nbsp;4672</td></tr>
            <tr><td>Fax:</td><td>&nbsp;&nbsp;&nbsp;</td><td>(+46)-8-15 3679</td></tr>
          </table>
          <br><br><br>
    </td></tr>
  </table>

  </table>

{% endblock %}

